README.md

```diff
@@ -1,2 +1,50 @@
+
+Settings will send you to the settings menu, where you change different options. Note: The settings menu is a little glitchy at the moment, and settings don't save past your current play session. For more consistency and permanence, go into the Pulsarc folder, and find "config.ini" Here you can see different options you can change. Comments have been included in the release version to help explain each option. You'll need to restart Pulsarc after saving any changes made in the config file.
+
+Currently Multiplayer and Editor are disabled. As these are developed and added to the game they will be enabled.
+
+And Quit shuts the game down.
+
+## How to Add Maps
+Right now releases comes with one map to play, but currently there's no editor or map-hosting service to get more. However, Pulsarc can convert osu!mania or Intralism maps into a pulsarc beatmap for you to play.
+To convert a map, find and open ``config.ini`` in your text editor of choice. Put "Intralism" or "Mania" after ``Game = ``. Find the path to the __folder__ of the map you want to convert. Copy the path address and then paste it after ``Path = ``. If you're converting an Intralism map, and want to import a specific background image from teh map, copy and paste the image name in front of ``BGImage = `` Save config.ini. You do not need to restart Pulsarc if it's currently running. In the Pulsarc Song Select screen, press the convert button (Default is "C"). If you gave the correct path, the map data should be converted to a new beatmap folder in the /Songs folder, and the Song Select screen should update to include the new map.
+
+## Build and Run
+To build and run Pulsarc, follow these steps:
+- Install the [.NET Core 2.2 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.2)
+- Clone the Pulsarc repository and its submodules ``git clone --recurse-submodules https://github.com/PulsarcGame/Pulsarc.git``
+- Build & Run Pulsarc with ``dotnet run --project Pulsarc``
+
+## Contributing
+If you are interested in contributing, it is strongly recommended that you join us on our [Discord Server](https://discord.gg/SYfpvfJ). Here you can chat in real time with other contributors, testers, and other community members. 
+
+It is also useful if you talk about a feature you want to develop on Discord or open a new issue before development starts. We will discuss whether it's something we want in Pulsarc.
+
+You can make a contribution through a pull request. Don't make requests for the master branch, use the [develop branch](https://github.com/PulsarcGame/Pulsarc). Please have one branch/pull request per feature, and keep pull requests short and sweet. We will let you know if we want to discuss more, or you can bring up a discussion on the Discord server.
+
+When contributing, please follow our (code style)[https://github.com/PulsarcGame/Pulsarc/blob/develop/CODESTYLE.md]. If you have any issues with the code style feel free to discuss on the issues board or our Discord server.
```

Comment:
I'm pretty sure you can just put `CODESTYLE.md` instead of entire link, since it's located in same repo. would shorten the links quite a bit. Same with `LICENSE` file below